keyboards/projectd/75/projectd_75_iso/info.json

```diff
@@ -0,0 +1,275 @@
+            { "flags": 4, "matrix": [2, 2], "x": 27.5, "y": 32.5 },
+            { "flags": 4, "matrix": [3, 2], "x": 37.5, "y": 32.5 },
+            { "flags": 4, "matrix": [4, 2], "x": 47.5, "y": 32.5 },
+            { "flags": 4, "matrix": [4, 3], "x": 57.5, "y": 32.5 },
+            { "flags": 4, "matrix": [5, 3], "x": 67.5, "y": 32.5 },
+            { "flags": 4, "matrix": [5, 2], "x": 77.5, "y": 32.5 },
+            { "flags": 4, "matrix": [6, 2], "x": 87.5, "y": 32.5 },
+            { "flags": 4, "matrix": [7, 2], "x": 97.5, "y": 32.5 },
+            { "flags": 4, "matrix": [8, 2], "x": 107.5, "y": 32.5 },
+            { "flags": 4, "matrix": [8, 3], "x": 117.5, "y": 32.5 },
+            { "flags": 4, "matrix": [10, 4], "x": 127.5, "y": 32.5 },
+            { "flags": 4, "matrix": [2, 5], "x": 145, "y": 32.5 },
+
+            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 42.5 },
+            { "flags": 4, "matrix": [0, 1], "x": 12.5, "y": 42.5 },
+            { "flags": 4, "matrix": [1, 4], "x": 22.5, "y": 42.5 },
+            { "flags": 4, "matrix": [2, 4], "x": 32.5, "y": 42.5 },
+            { "flags": 4, "matrix": [3, 4], "x": 42.5, "y": 42.5 },
+            { "flags": 4, "matrix": [4, 4], "x": 52.5, "y": 42.5 },
+            { "flags": 4, "matrix": [4, 5], "x": 62.5, "y": 42.5 },
+            { "flags": 4, "matrix": [5, 5], "x": 72.5, "y": 42.5 },
+            { "flags": 4, "matrix": [5, 4], "x": 82.5, "y": 42.5 },
+            { "flags": 4, "matrix": [6, 4], "x": 92.5, "y": 42.5 },
+            { "flags": 4, "matrix": [7, 4], "x": 102.5, "y": 42.5 },
+            { "flags": 4, "matrix": [8, 5], "x": 112.5, "y": 42.5 },
+            { "flags": 4, "matrix": [9, 1], "x": 122.5, "y": 42.5 },
+            { "flags": 4, "matrix": [3, 5], "x": 142.5, "y": 45 },
+            { "flags": 4, "matrix": [7, 5], "x": 155, "y": 42.5 },
+
+            { "flags": 4, "matrix": [0, 6], "x": 0, "y": 52.5 },
+            { "flags": 4, "matrix": [9, 0], "x": 12.5, "y": 52.5 },
+            { "flags": 4, "matrix": [9, 3], "x": 25, "y": 52.5 },
+            { "flags": 4, "x": 61.5, "y": 52.5 },
+            { "flags": 4, "x": 62.5, "y": 52.5 },
+            { "flags": 4, "matrix": [9, 4], "x": 65, "y": 52.5 },
+            { "flags": 4, "x": 67.5, "y": 52.5 },
+            { "flags": 4, "x": 68.5, "y": 52.5 },
+            { "flags": 4, "matrix": [9, 5], "x": 100, "y": 52.5 },
+            { "flags": 4, "matrix": [9, 2], "x": 110, "y": 52.5 },
+            { "flags": 4, "matrix": [0, 4], "x": 120, "y": 52.5 },
+            { "flags": 4, "matrix": [0, 3], "x": 132.5, "y": 52.5 },
+            { "flags": 4, "matrix": [7, 3], "x": 142.5, "y": 52.5 },
+            { "flags": 4, "matrix": [0, 5], "x": 152.5, "y": 52.5 }
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0x0011",
+        "vid": "0x3233"
+    },
+    "layouts": {
+        "LAYOUT_75_iso": {
```

Comment:
would you be able to correct key spacing/width?
![layout_incorrect](https://github.com/qmk/qmk_firmware/assets/75046609/6e793c30-70e0-41b3-b146-73d36a4826bf)